src/react/Alert/index.tsx

```diff
@@ -11,6 +11,8 @@ type AlertProps = {
 }
 
 const Alert = ({text, type, isDisabled, isVisible, onClick}: AlertProps) => {
```

Comment:
`isDisabled` is not used in the component so I think it should be removed here.

Reply:
@phongnhat19 
I think if we delete `isDisabled` immediately, people who specify this param like following in old ui-component will get compile error when update ui-component and use latest component.

`var label = new kintoneUIComponent.Label({text: 'label', isDisable: 'false'});`

So it is better to delete `isDisabled` when we update major version of ui-component.
What do you think ?

Reply:
@youtr-s 
We remove `isDisabled` from destructed props of React, so using it in JS (as your sample) won't be affected.